Backend/models/sellerDetails.model.js

```diff
@@ -2,35 +2,32 @@ import mongoose, { Schema } from "mongoose";
 import mongooseAggregatePaginate from "mongoose-aggregate-paginate-v2";
 
 const sellerDetailsSchema = new Schema({
-
     email: {
-        type: Schema.Types.ObjectId,
-        ref: "Seller"
```

Comment:
Email is supposed to be a reference only.

Reply:
required modifications are done.